frontend/src/bundles/chat/components/chat-footer/chat-footer.tsx

```diff
@@ -0,0 +1,18 @@
+import { Box } from '~/bundles/common/components/components.js';
+
+import { type GenerateTextRequestDto } from '../../types/types.js';
+import { ChatForm } from '../chat-form/chat-form.js';
+
+type Properties = {
+    onSendMessage: (payload: GenerateTextRequestDto) => void;
+};
+
+const ChatFooter: React.FC<Properties> = ({ onSendMessage }) => {
+    return (
+        <Box>
```

Comment:
I am not sure if we need a separate component just to wrap ChatForm

Reply:
ok at first thought that maybe the footer could have others things to make a more complex chat in the near future, but we should create this other component when is required as you mention, thanks